### PR TITLE
Content Edits for MySQL Workshop

### PR DESCRIPTION
**Pull request by danielhelfand**
This pull request touches up content and language to help with user experience while going through the mysql-operator workshop. These are suggestions based on my experience of going through the mysql workshop.

I encountered no technical issues while going through the workshop though.

### CODE REVIEW

File: workshop/content/exercises/01-operator-prerequisites.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ NextPage: 02-create-cluster
 
 The MySQL operator is installed into a project to monitor requests to create a MySQL cluster. The operator can only be deployed and setup by a cluster admin of the OpenShift cluster.
 
-In this workshop environment, the MySQL operator has been pre-installed into your project ready for use. The user you are running the workshop as, has also been delegated the appropriate roles to allow it to create a MySQL cluster.
+In this workshop environment, the MySQL operator has been pre-installed into your project ready for use. The user you are running the workshop as has also been delegated the appropriate roles to allow it to create a MySQL cluster.
```
Comment (joshix) [inline workshop/content/exercises/01-operator-prerequisites.md]: This comma is unnecessary. Read the sentence aloud; one doesn't pause between "as" and "has". The sentences could be rendered more clearly, if you want to make some change:
"The MySQL cluster operator has been installed and is ready to use. Your user has been granted the appropriate permissions to create a managed MySQL database cluster."

**Review by joshix — CHANGES_REQUESTED** [review]
+1 on adding the sample output.

File: workshop/content/exercises/01-operator-prerequisites.md (hunk 1 above)
Comment (danielhelfand) [reply workshop/content/exercises/01-operator-prerequisites.md]: Thanks! I just pushed something up. Let me know if this reads a little bit clearer.

Comment (joshix) [reply workshop/content/exercises/01-operator-prerequisites.md]: LGTM. Thank you. :)

**Pull request merged**